[2025-03-11T09:30:51.713+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-11T09:30:51.973+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.stores manual__2025-03-11T09:30:31.306143+00:00 [queued]>
[2025-03-11T09:30:52.163+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.stores manual__2025-03-11T09:30:31.306143+00:00 [queued]>
[2025-03-11T09:30:52.165+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-11T09:30:52.422+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): stores> on 2025-03-11 09:30:31.306143+00:00
[2025-03-11T09:30:52.494+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8525) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-11T09:30:52.507+0000] {standard_task_runner.py:72} INFO - Started process 8537 to run task
[2025-03-11T09:30:52.506+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'stores', 'manual__2025-03-11T09:30:31.306143+00:00', '--job-id', '253', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmpjz7npj4_']
[2025-03-11T09:30:52.519+0000] {standard_task_runner.py:105} INFO - Job 253: Subtask stores
[2025-03-11T09:30:54.637+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.stores manual__2025-03-11T09:30:31.306143+00:00 [running]> on host efacfdd0be48
[2025-03-11T09:30:55.517+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='stores' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T09:30:31.306143+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T09:30:31.306143+00:00'
[2025-03-11T09:30:55.557+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-11T09:30:55.563+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-11T09:30:55.564+0000] {logging_mixin.py:190} INFO - Current task name:stores state:running start_date:2025-03-11 09:30:51.985993+00:00
[2025-03-11T09:30:55.577+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-11T09:30:55.581+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-11T09:30:55.691+0000] {logging_mixin.py:190} INFO - Validation successful. Found 3 store records.
[2025-03-11T09:30:55.703+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-11T09:30:55.841+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-11T09:30:55.852+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=stores, run_id=manual__2025-03-11T09:30:31.306143+00:00, execution_date=20250311T093031, start_date=20250311T093051, end_date=20250311T093055
[2025-03-11T09:30:56.073+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-11T09:30:56.096+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-11T09:30:56.113+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-11 09:30:31.414980+00:00
[2025-03-11T09:30:56.123+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-11T09:30:56.304+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-11T09:30:57.415+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-11T09:30:57.449+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-11T09:30:57.528+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
